cypress/e2e/test73588.cy.ts

```diff
@@ -0,0 +1,26 @@
+
+
+describe('DropDown Select Test', ()=>{
+   it('select combobox', ()=> {
+       cy.visit('https://obstaclecourse.tricentis.com/Obstacles/73588/');
+       cy.get('#clickme').should('exist');
+       cy.get('#clickme').click();
+       cy.wait(1000);
+
+       cy.get('#randomtext').should('exist');
+       // cy.get('#randomtext').should('not.be.empty');
+
+       cy.get('#randomtext').invoke('val').then((val) => {
+           console.log(val);
+           cy.get('#selectlink').children().each(($el) => {
+                   if ($el.text() === val) {
+                       cy.get('#selectlink').select($el.text());
+                   }
+               }
+           );
+           cy.get('#submitanswer').click();
+       });
+   });
+});
+
+
```

Comment:
After all changes, you should finally write code as below:

it('73588', ()=> {
      cy.visit('https://obstaclecourse.tricentis.com/Obstacles/73588/')
      cy.get('#clickme').click()
      cy.get('#randomtext').should('not.have.value', '')
      cy.get('#randomtext').invoke('val').then((val)=>{
        cy.get('#selectlink').select(val)
      })
      cy.get('#submitanswer').click()
      cy.contains("Good job!")
    })